docs/users-guide/models.md

```diff
@@ -110,19 +110,32 @@ Just like with a question, admins can verify models. Verifying a model will give
 
 _Currently available for PostgreSQL, MySQL, and Redshift_.
 
-You can enable caching for models so that loading questions based on those models is significantly faster. Instead of running the query that creates the model fresh each time, Metabase will periodically run the query and store the results in a table in your data warehouse, so that whenever someone loads the model (or a question based on that model), Metabase will simply fetch the saved results from the table in your data warehouse. To store cached results, Metabase creates a new schema in your data warehouse, and creates new tables in that schema to store the results of the queries that power the models.
+You can enable caching for models so that loading questions based on those models is significantly faster. Instead of running the query that creates the model fresh each time, Metabase will periodically run the query and store the results in a table in your data warehouse, so that whenever someone loads the model (or a question based on that model), Metabase will simply fetch the saved results from the table in your data warehouse.
 
-We recommend scheduling the cache to refresh on a frequency that makes sense with how often your source tables update with new data.
+To store cached results, Metabase creates a new schema in your data warehouse, and creates new tables in that schema to store the results of the queries that power the models. In order for Metabase to cache models in your database, the connection credentials you use to connect Metabase to your data warehouse need to be able to create new schemas in your data warehouse. So, for example, if you're connected to a PostgreSQL database, the Postgres user account that you use to connect Metabase to that Postgres database must have privileges to create new schemas in that Postgres database.
```

Comment:
There are two credentials options:
1. Privilege to create and manage schemas
2. Privilege to manage the specific schema (the schema name is displayed on the database page and must be created before turning model cache on) (this is more appropriate, as has less access than the first one)